### PR TITLE
ADDED BMI CALCULATOR 

### LINKED ISSUE #126: Bmi calculator in js
It is basically a bmi calculator using javascript.
There will be 2 inputs for weight and height and then after clicking on button the bmi will be calculated
It can be used in various gyms.
Plzz assign me this under lgmsoc

### PR DESCRIPTION
**Pull request by siddhi-244**
Fixes #126

### CODE REVIEW

**Review by Ayushparikh-code — APPROVED** [review]

**Review by harshita2216 — APPROVED** [review]

File: bmi-calculator/index.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+		<p class="text">Weight</p>
+		<input type="text" id="w">
+		<p id="result"></p>
+		<button id="btn" onclick="bmi()">Calculate</button>
+		<p id="info">Please enter height[cm] and weight [kg]</p>
+	</div>
+	<script type="text/javascript" src="script.js"></script>
+
+</body>
+</html>
```
Comment (chicken-biryani) [inline bmi-calculator/index.html:21]: Add a new line at the end of file.

File: bmi-calculator/script.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+function bmi(){
+	let h=document.getElementById("h").value;
+	let w=document.getElementById("w").value;
+	let ans=w/(h/100*h/100);
+	let bmio=(ans.toFixed(2));
+	document.getElementById("result").innerHTML="Your BMI is "+bmio;
+}
```
Comment (chicken-biryani) [inline bmi-calculator/script.js:7]: Add a new line at the end of file

File: bmi-calculator/style.css [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+	cursor: pointer;
+}
+#btn:hover{
+	box-shadow: 1px 1px 10px  #341f97; 
+
+}
+#info{
+	font-size: 12px;
+	
+}
```
Comment (chicken-biryani) [inline bmi-calculator/style.css:67]: Add a new line at the end of file

File: bmi-calculator/README.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+git clone https://github.com/Ayushparikh-code/Web-dev-mini-projects.git
+```
+
+- Go to the directory
+- Run the index.html file
+- Start Calculating!!
+--- 
+
+## Screenshots 
+![Screenshot (35)](https://user-images.githubusercontent.com/69195262/125057793-3d825f00-e0c7-11eb-8a87-a80ecf6603e1.png)
```
Comment (chicken-biryani) [inline bmi-calculator/README.md:29]: Add a new line at the end of file.

**Review by chicken-biryani — COMMENTED** [review]
Hey @siddhi-244 everything looks perfect. You've done a great job. Just add a new line at the end of each file and the PR is good to go. :)

**Comment by siddhi-244** [comment]
Done maam @chicken-biryani

**Comment by monizb** [comment]
Hi @chicken-biryani a LGMSOC21 team member here, would you be so kind as to add the LGMSOC21 labels in all these PRs at the earliest:

https://github.com/Ayushparikh-code/Web-dev-mini-projects/pull/198
https://github.com/Ayushparikh-code/Web-dev-mini-projects/pull/162
https://github.com/Ayushparikh-code/Web-dev-mini-projects/pull/158
https://github.com/Ayushparikh-code/Web-dev-mini-projects/pull/120
https://github.com/Ayushparikh-code/Web-dev-mini-projects/pull/118

**Pull request merged**